README.md

```diff
@@ -1,4 +1,174 @@
+
+###### Output
+
+* **`evalscripts`**: *list of dicts*
+
+    Returns a list of dicts containing the `Evalscript` objects. Every element consists of:
+    - `invalid_node_id`: Id of the first invalid node after a supported subgraph. The output of the associated evalscript should be the input of the node. If it is `None`, the entire graph is valid.
+
+    - `evalscript`: instance of and `Evalscript` object that generates a an evalscript for a valid subgraph from `load_collection` to the node with id `invalid_node_id`. 
+
+
+#### Evalscript
+
+###### Constructor parameters
+
+* **`input_bands`**: *list*
+
+    List of bands to be imported. See [docs](https://docs.sentinel-hub.com/api/latest/evalscript/v3/#input-object-properties).
+
+* **`nodes`**: *list*
+
+    List of `Node` objects that constitute the valid process (sub)graph.
+
+* **`initial_data_name`**: *str*
+
+    The id of the initial `load_collection` node that loads the data.
+
+* **`n_output_bands`**: *int, optional*. Default: `1`
+
+    Number of output bands in the evalscript. This can be set if the value is known beforehand. See [docs](https://docs.sentinel-hub.com/api/latest/evalscript/v3/#input-object-properties).
+
+* **`sample_type`**: *str, optional*. Default: `FLOAT32`
+
+    Desired `sampleType` of the output raster. See [possible values](https://docs.sentinel-hub.com/api/latest/evalscript/v3/#sampletype).
+
+
+* **`units`**: *str, optional*. Default: `None`
+
+    Units used by all the bands in the evalscript. If `None`, `units` evalscript parameter isn't set and default units for each band are used. See [docs](https://docs.sentinel-hub.com/api/latest/evalscript/v3/#input-object-properties).
+
+* **`mosaicking`**: *str, optional*. Default: `ORBIT`
+
+    Works with multi-temporal data by default. See [possible values](https://docs.sentinel-hub.com/api/latest/evalscript/v3/#mosaicking).
+
+* **`bands_dimension_name`**: *str, optional*. Default: `bands`
+
+    Name of the default dimension of type `bands` of the datacube, as set in `load_collection` and referred to in the process graph.
+
+* **`temporal_dimension_name`**: *str, optional*. Default: `t`
+
+    Name of the default dimension of type `temporal` of the datacube, as set in `load_collection` and referred to in the process graph.
+
+* **`datacube_definition_directory`**: *str, optional*. Default: `javascript_datacube`
+
+    Relative path to the directory with the javascript implemetations of the processes.
+
+* **`output_dimensions`**: *list of dicts or None, optional*. Default: `None`
+
+    Information about the dimensions in the output datacube. This can be set if the value is known beforehand. Each element contains:
+    - `name`: name of the dimension,
+    - `size`: size (length) of the dimension,
+    - `original_temporal`, optional: boolean, should be `True` if this is the temporal dimension generated in the initial `load_collection` node.
+
+###### Methods
+
+* **`write`** ():
```

Comment:
Seems like `()` should be in the same font as the name of the method.

FFTR if I'm wrong.

Reply:
Looks good now.